src/viam/services/slam/client.py

```diff
@@ -38,8 +38,8 @@ async def get_position(self, *, timeout: Optional[float] = None) -> Pose:
         response: GetPositionResponse = await self.client.GetPosition(request, timeout=timeout)
         return response.pose
 
-    async def get_point_cloud_map(self, *, timeout: Optional[float] = None) -> List[bytes]:
-        request = GetPointCloudMapRequest(name=self.name)
+    async def get_point_cloud_map(self, return_edited_map: Optional[bool] = None, *, timeout: Optional[float] = None) -> List[bytes]:
```

Comment:
This should also be just a `bool` defaulted to `False`, rather than an `Optional[bool] = None` (similar to the change you made in `slam.py`)

Reply:
ah my bad, got the one in the test but I forgot this one